syntaxes/hcl.tmGrammar.json

```diff
@@ -98,8 +98,13 @@
         "2": {
           "patterns": [
             {
-              "match": "[\\\"\\-\\w]+",
-              "comment": "Block label",
+              "match": "\\\"[^\\\"\\r\\n]*\\\"",
```

Comment:
```suggestion
              "match": "\\\"[^\\\"\\r\\n]+\\\"",
```

Reply:
This file is autogenerated, so we can use the comment in the YAML file